src/vcpkg/install.cpp

```diff
@@ -602,8 +602,8 @@ namespace vcpkg::Install
         const auto& message = usage.message;
         if (!message.empty())
         {
-            auto existing = printed_usage.find(message);
-            if (existing == printed_usage.end())
```

Comment:
I don't get it, how is the new code different from the old one? 😅

Reply:
New one
```c++
            auto existing = printed_usages.lower_bound(message);
            if (existing == printed_usages.end() || *existing != message)
```

Reply:
I'm not sure what this is pointing at since it isn't even in the PR?